plugins/Morpheus/stylesheets/theme-advanced.less

```diff
@@ -0,0 +1,17 @@
+@theme-color-background-base:          #fff;
+@theme-color-background-tinyContrast:  #f2f2f2;
+@theme-color-background-lowContrast:   @color-silver-l80;
+@theme-color-background-contrast:      #5F5A60;
+@theme-color-background-highContrast:  #202020;
+
+@theme-color-border: @color-silver-l80;
+
+@theme-color-menu-contrast-text: @theme-color-text-lighter;
```

Comment:
Not sure where I mentioned it, eg in https://github.com/piwik/piwik/issues/7876#issuecomment-100704490 I mentioned to put the menu and widget colors in `theme.less`. They are very important for theming. The `@theme-color-background-*` variables should be still in `theme-advanced` as they are "complicated" (whch I also mentioned here https://github.com/piwik/piwik/pull/7793#issuecomment-98549220 ). 

I know they default to other variables but we want users to be able to just specifically change those widget and menu colors without affecting anything else to keep it as easy as possible. Just by changing those 5 menu and widget variables they will be able to create a nice theme according to their CI.